lib/depject/profile/html/preview.js

```diff
@@ -4,9 +4,11 @@ const map = require('mutant/map')
 const when = require('mutant/when')
 const computed = require('mutant/computed')
 const send = require('mutant/send')
+const removeMd = require('remove-markdown')
```

Comment:
Instead of this, and `shortenDescription`, I think you should be able to use https://github.com/ssbc/patchwork/blob/master/lib/depject/message/async/name.js#L53 instead. It's what we use to create the text for e.g. links to forks and references.